.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Log in to Docker Hub to avoid rate limiting
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.FARCASTERXYZ_DOCKER_HUB_USER }}
+          username: ${{ vars.FARCASTERXYZ_DOCKER_HUB_USER }}
```

Comment:
intentional?

Reply:
yeah, not sure when this changed but this wasn't actually a secret in the repo, it's a var.